### PR TITLE
Fix: Action column gone

### LINKED ISSUE #644: Actions column gone
**What happened**:
`ACTIONS` column disappeared

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:
When navigating to `ALL_PROJECTS` and then back to a project, the `ACTIONS` column is gone

**Anything else we need to know**:

**Environment**:

### PR DESCRIPTION
**Pull request by petersutter**
**What this PR does / why we need it**:
Action column stays hidden if it was set to hidden once. This undesired behaviour is fixed with this PR

**Which issue(s) this PR fixes**:
Fixes #644 

**Special notes for your reviewer**:

**Release note**:

```improvement user
Fixed: Action column was hidden when navigating from `ALL_PROJECTS` to a project
```

### CODE REVIEW

File: frontend/src/pages/ShootList.vue [hunk 1]
```diff
@@ -274,20 +274,18 @@ export default {
...
-          this.hideHeaderIfNotValue(header, this.gitHubRepoUrl)
+          this.hideHeaderIfNotValue(header, this.gitHubRepoUrl && this.isAdmin)
         }
         if (header.value === 'actions') {
           this.hideHeaderIfNotValue(header, this.canDeleteShoots || this.canGetSecrets)
         }
       }
     },
     hideHeaderIfNotValue (header, value) {
-      if (!value) {
```
Comment (grolu) [inline frontend/src/pages/ShootList.vue]: Let's discuss this

**Comment by petersutter** [comment]
@grolu thank you for the commit, lgtm

File: frontend/src/pages/ShootList.vue [hunk 2]
```diff
@@ -273,21 +275,22 @@ export default {
     },
     hideNotAvailableColumns () {
```
Comment (holgerkoser) [inline frontend/src/pages/ShootList.vue]: rename to `setColumnVisibility`

File: frontend/src/pages/ShootList.vue [hunk 3]
```diff
@@ -273,21 +275,22 @@ export default {
...
+          case 'journalLabels':
+          case 'journal':
+            this.hideHeaderIfNotValue(header, this.gitHubRepoUrl && this.isAdmin)
+            break
+          case 'actions':
+            this.hideHeaderIfNotValue(header, this.canDeleteShoots || this.canGetSecrets)
+            break
+          case 'project':
+            this.hideHeaderIfNotValue(header, !this.projectScope)
+            break
```
Comment (holgerkoser) [inline frontend/src/pages/ShootList.vue]:
```suggestion
          case 'journal':
            header.hidden = !(this.gitHubRepoUrl && this.isAdmin)
            break
          case 'actions':
            header.hidden = !(this.canDeleteShoots || this.canGetSecrets)
            break
          case 'project':
            header.hidden = !!this.projectScope
            break
```

File: frontend/src/pages/ShootList.vue [hunk 4]
```diff
@@ -273,21 +275,22 @@ export default {
...
         }
       }
     },
     hideHeaderIfNotValue (header, value) {
-      if (!value) {
-        header.hidden = true
-      }
+      header.hidden = !value
```
Comment (holgerkoser) [inline frontend/src/pages/ShootList.vue]: delete method

File: frontend/src/pages/ShootList.vue (hunk 2 above)
Comment (petersutter) [reply frontend/src/pages/ShootList.vue]: ✅

File: frontend/src/pages/ShootList.vue (hunk 3 above)
Comment (petersutter) [reply frontend/src/pages/ShootList.vue]: ✅

**Pull request merged**